### PR TITLE
Fix (skip) 17490 repro

### PR DESCRIPTION
**Pull request by dragonsahead**
Seems that we removed the finder

### CODE REVIEW

**Review by rlotun — APPROVED** [review]

File: frontend/test/metabase/scenarios/native-filters/reproductions/17490.cy.spec.js [hunk 1]
```diff
@@ -31,12 +31,10 @@ describe("issue 17490", () => {
      * Cypress fails to click any element that is not "actionable" (for example - when it's covered).
      * In other words, the `.click()` part is essential for this repro to work. Don't remove it.
      */
-    cy.findByPlaceholderText("Find...")
+    cy.findByDisplayValue("Orders");
```
Comment (rlotun) [inline frontend/test/metabase/scenarios/native-filters/reproductions/17490.cy.spec.js]: actually there is a trailing semicolon

**Review by rlotun — CHANGES_REQUESTED** [review]

File: frontend/test/metabase/scenarios/native-filters/reproductions/17490.cy.spec.js (hunk 1 above)
Comment (paoliniluis) [reply frontend/test/metabase/scenarios/native-filters/reproductions/17490.cy.spec.js]: yup, just removed it and sent to test again, thanks for the review!

**Review by rlotun — APPROVED** [review]

**Comment by flamber** [comment]
This test isn't valid anymore - at least not with the current changed. Better just to revert and skip it with a comment.

**Comment by paoliniluis** [comment]
just reversed and skipped

**Pull request merged**